spec/30-processing-model.md

```diff
@@ -0,0 +1,96 @@
+# Processing Model
+
+This section describes the behavior of a tracing system when a new request is received.
+As stated above, tracing systems can decide for each request whether they want to
+just _forward_ or _participate_ in a trace context. Additionally, when participating,
+a system needs to be able to start or restart a trace.
+
+## Processing Model in Forwarding Mode
+
+In forwarding mode, a tracing system decides to forward tracing information
+but does not modify the tracecontext headers.
+
+1. The tracing system receives a `traceparent` header and may receive a `tracecontext`.
```

Comment:
You wrote `tracecontext`, but don't you mean `tracestate`? There are multiple such occurrences in the following text.